Mathlib/Data/Option/Defs.lean

```diff
@@ -16,6 +16,11 @@ Other basic operations on `Option` are defined in the core library.
 
 namespace Option
 
+/-- Inhabited `get` function. Returns `a` if the input is `some a`, otherwise returns `default`. -/
+@[reducible] def iget [Inhabited α] : Option α → α
+| (some x) => x
+| none     => default
+
```

Comment:
Would it make sense to do the complete port of Data/Option/Defs first?

Reply:
Okay, will do!